cosmo_tester/test_suites/image_based_tests/cli_test.py

```diff
@@ -312,5 +313,5 @@ def _copy_terraform_files(self):
             'terraform/scripts/osx-cli-test.sh'),
             self.tmpdir / 'scripts/osx-cli-test.sh')
 
-    def run_test(self):
-        super(_OSXCliPackageTester, self).run_test()
+    def run_test(self, inputs):
+        super(_OSXCliPackageTester, self).run_test(inputs)
```

Comment:
this method can be removed, because all it does is run super (and hide the return value (which is none anyway))